src/utils/utils-contour.js

```diff
@@ -61,7 +65,7 @@ export const buildContourSQL = (state, mapBounds, isPolygons = false) => {
   const contourParamsSQL = buildParamsSQL(contourParams)
 
   const geometryColumn = isPolygons ? "contour_polygons" : "contour_lines"
-  const contourLineCase = `CASE mod(cast(contour_values as int), ${major_contour_interval}) WHEN 0 THEN 1 ELSE 0 END as is_major `
+  const contourLineCase = `CASE mod(cast(contour_values as int), ${major_contour_interval}) WHEN 0 THEN 1 ELSE 0 END as ${encoding.color.field} `
```

Comment:
Also, what happens if `encoding.color` doesn't exist? i.e. the user wants to use a flat color for all the contours, but wants them sized differently based on major/minor interval - so encoding.strokeWidth.field exists but not color.

Also, what happens if the user wants to color the contours by some other potential measure, like the length of the line? -- not sure we would have that kind of control initially, but it's possible a user may want to color / size by something other than the is_major distinction at some point. We would need to think about handling that.

Reply:
This should be handled by formatting changes above, this now reads from `data.interval.intervalTypeFieldName`, but let me know if you're looking for something else